doc/overview.md

```diff
@@ -0,0 +1,142 @@
+# APIcast Overview
+
+APIcast is an NGINX based API gateway used to integrate your internal and external API services with 3scale’s API Management Platform.
+
+Here you’ll learn more about deployment options, environments provided, and how to get started.
+
+## Deployment options
+
+You can use APIcast hosted or self-managed, in both cases, it needs connection to the rest of the 3scale API management platform:
+- **<a href="https://support.3scale.net/docs/deployment-options/apicast-cloud-gateway">APIcast hosted</a>**: 3scale hosts the gateway in the cloud. The API gateway is already deployed for you and it's limited to 50,000 calls per day.
+- **APIcast self-managed**: You can deploy APIcast wherever you want. To do so, download the json configuration file from API > Integration > Production or fetch it using the Account management API. The self-managed mode is the intended mode of operation for production environments. Here are a few recommended options to deploy your API gateway:
+  - <a href="https://support.3scale.net/docs/deployment-options/nginx-self-managed-setup">Local deploy</a>: Where the customer needs to deploy a supported version of Nginx/ OpenResty and use the 'GA' tagged release from the [APIcast repository](https://github.com/3scale/apicast/releases).
+  - <a href="https://support.3scale.net/guides/infrastructure/docker-openshift">OpenShift</a>: Where the customer deploys on a supported version of OpenShift a released and unmodified OpenShift template for APIcast, which will run the official Docker images.
+  - Docker: Where the customer runs a dockerized image downloaded from the Red Hat Docker Registry, using a supported version of Docker.
+
+## Environments
+
+By default, when you create a 3scale account, you get APIcast **hosted** in two different environments:
+
+- **Staging**: It’s meant to be used only while configuring and testing your API integration. When you have confirmed that your setup is working as expected, then you can choose to deploy it to the production environment.
+- **Production**: Limited 50,000 calls per day and supports the following out-of-the-box authentication options: App key, app key and app id pair.
+
+## Getting API traffic in 1 minute
+
+Follow the next steps to configure your API gateway in no time:
+
+### 1. Declare your private base URL

```

Comment:
I think this will make more sense on the support portal with screenshots, but here we could provide a way to get to the Integration Page.